apps/client/src/components/Conclusion/ContactConclusion.tsx

```diff
@@ -0,0 +1,35 @@
+import { Heading } from "@datapunt/asc-ui";
+import React from "react";
+import { isIE, isMobile } from "react-device-detect";
+
+import { HideForPrint, PrintButton } from "../../atoms";
+import { sections } from "../../config/matomo";
+import Markdown from "../Markdown";
+
+type Props = {
+  contactConclusion: { [key: string]: string };
```

Comment:
Why did you go for `{ [key: string]: string }`?
Maybe specify `title` and `description` here?